getting-started/frames.ipynb

```diff
@@ -111,7 +111,7 @@
    "outputs": [],
    "source": [
     "# Relative path to the NoSQL table within the parent platform data container\n",
-    "table = os.path.join(os.getenv(\"V3IO_USERNAME\") + \"/examples/bank\")\n",
+    "table = os.path.join(os.getenv(\"V3IO_USERNAME\"), \"/examples/bank\")\n",
```

Comment:
you dont need the slash before the `examples/bank` same for the other place you changed

Reply:
@talIguaz missed 3 instances ... => FIXED (commit a409c9a).